### ISSUE TITLE
Menu opened with 'hold' disappears on releasing the mouse button.

### ISSUE THREAD

**Issue by jgosmann** [nengo/nengo_gui#338]

**Comment by celiasmith** [comment]
Is this an issue on touchscreens as well?

**Comment by cchan987** [comment]
Hmm, I can't replicate but similar to what you said I found:
1) right click and hold (menu appears)
2) Move mouse while holding (menu disappears)
3) on right-release (menu still hidden)

could the mouse movement be causing the issue? perhaps this is a mac issue?

**Comment by celiasmith** [comment]
It's not a mac issue, as it happens to Terry.  However, this probably shouldn't be in the milestone because no one is using touchscreens, and there are a million other touchscreen issues.

**Comment by jgosmann** [comment]
Yeah, I guess it doesn't need to be in the milestone. Though, it probably is a simple fix.

**Comment by cchan987** [comment]
Is this still an issue? Unable to reproduce.

**Comment by jgosmann** [comment]
Yes, still in issue. Happens in Firefox, Chrome, Linux, OS X, with track pad, and mouse. No mouse movement involved.
